src/Datadog.Trace.ServiceFabric/Propagators/ServiceFabricContextPropagator.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.ServiceFabric.Services.Remoting.V2;
+
+namespace Datadog.Trace.ServiceFabric.Propagators
+{
+    internal abstract class ServiceFabricContextPropagator
```

Comment:
any reason why an abstract class instead of an interface? performance?

Reply:
just a refactoring error.